ecommerce/pols/templates/oscar/basket/partials/hosted_checkout_basket.html

```diff
@@ -80,9 +80,15 @@
                             {% if line_data.line.has_discount %}
                                 <div class="discount">
                                     <div class="benefit">
-                                        {% blocktrans with benefit_value=line_data.benefit_value %}
-                                            {{ benefit_value }} off
-                                        {% endblocktrans %}
+                                        {% if "$" in line_data.benefit_value %}
```

Comment:
Could you add a comment related to what you're doing here? @Jacatove Add it as a template comment, not an HTML comment.